docs/en/latest/deployments/minikube.md

```diff
@@ -41,7 +40,7 @@ kubectl create ns ingress-apisix
 helm install apisix apisix/apisix \
   --set gateway.type=NodePort \
   --set ingress-controller.enabled=true \
-  --namespace ingress-apisix
+  --namespace ingress-apisix \
   --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
```

Comment:
Does it already exist? @tao12345666333

Reply:
yep. it was introduced by https://github.com/apache/apisix-ingress-controller/pull/747  I forgot